### PR TITLE
[iPhoneX] Gets onboarding to work with iPhone X

### PR DESCRIPTION
**Pull request by orta**
Adds support for taking into account the top and bottom margins on an iPhone X

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">No CHANGELOG changes made</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR is classed as Work in Progress</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Needs testing on a Phone if change is non-trivial</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by orta** [comment]
screen shot 2018-01-24 at 1 06 42 pm

Onboarding with black top, it'll animate down with the first view controller in the workflow

screen shot 2018-01-24 at 1 06 44 pm

Auctions modal x

screen shot 2018-01-24 at 1 46 40 pm

Everyone gets the new **round** live lot CLA

screen shot 2018-01-24 at 12 44 29 pm

Had a fun bug at one point, thought this looked pretty

screen shot 2018-01-24 at 11 07 52 am

File: Artsy/View_Controllers/Live_Auctions/Views/LiveAuctionCurrentLotView.swift [hunk 1]
```diff
@@ -43,6 +44,8 @@ class LiveAuctionCurrentLotView: UIButton {
         thumbnailView.alignLeadingEdge(withView: self, predicate: "10")
         thumbnailView.constrainWidth("38", height: "38")
         thumbnailView.alignCenterY(withView: self, predicate: "0")
+        thumbnailView.layer.cornerRadius = 5
+        thumbnailView.layer.masksToBounds = true;
```
Comment (ashfurrow) [inline Artsy/View_Controllers/Live_Auctions/Views/LiveAuctionCurrentLotView.swift]: Curious why these were needed?

Comment (ashfurrow) [reply Artsy/View_Controllers/Live_Auctions/Views/LiveAuctionCurrentLotView.swift]: Oh I see. Hmm. Okay.

**Pull request merged**